classes/Entities/UserLastaccess.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Avado\MoodleAbstractionLibrary\Entities;
+
+/**
+ * Class UserLastaccess
+ * @package Avado\MoodleAbstractionLibrary\Entities
+ */
+class UserLastaccess extends BaseModel
+{
+    /**
+     * @var string
+     */
+    protected $table = 'user_lastaccess';
+}
+
```

Comment:
Can you remove one of these extra lines please?

Reply:
done. please check